command/agent/http.go

```diff
@@ -355,6 +360,54 @@ func (s *HTTPServer) wrap(handler func(resp http.ResponseWriter, req *http.Reque
 	return f
 }
 
+// wrapNonJSON is used to wrap functions returning non JSON
+// serializeable data to make them more convenient. It is primarily
+// responsible for setting nomad headers and logging.
```

Comment:
Like this function.  Should clarify that handler function ought to set Content-Type header?